### PR TITLE
initial config export - cvocni

### PR DESCRIPTION
**Pull request by AdamMcFet**

### CODE REVIEW

File: .platform/routes.yaml [hunk 1]
```diff
@@ -54,6 +54,14 @@
 'https://hiaredressni.uk/':
     type: redirect
     to: 'https://www.hiaredressni.uk/'
+'https://www.cvocni/':
```
Comment (soda736) [inline .platform/routes.yaml:57]: This should be cvocni.org?

File: .platform/routes.yaml [hunk 2]
```diff
@@ -54,6 +54,14 @@
...
+    type: upstream
+    upstream: 'unity_svr_2:http'
+    cache:
+        enabled: 'false'
+'https://cvocni/':
+    type: redirect
+    to: 'https://www.cvocni/'
```
Comment (soda736) [inline .platform/routes.yaml:64]: And here too cvocni.org?

File: project/project.yml [hunk 3]
```diff
@@ -72,3 +72,11 @@ sites:
         database: hiaredressni
         status: development
         default: false
+    cvocni:
+        name: 'Commission for victims of crime'
+        url: cvocni
```
Comment (soda736) [inline project/project.yml:77]: The site URL needs to be cvocni.org

File: project/config/cvocni/config/unity_internal_link_checker.linksettings.yml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+site_url_list: "https://www.cvocni.org/\r\nhttps://cvocni.org/\r\nhttp://www.cvocni.org/\r\nhttp://cvocni.org/"
```
Comment (soda736) [inline project/config/cvocni/config/unity_internal_link_checker.linksettings.yml:1]: We don't need these URL's anymore as there is background code to add them all in. I need to update the new site documentation, sorry.

**Review by soda736 — CHANGES_REQUESTED** [review]

**Review by DuttonMa — APPROVED** [review]

**Review by soda736 — APPROVED** [review]

**Pull request merged**